### PR TITLE
Best practices

### PR DESCRIPTION
**Pull request by medhamsh**
- Passing through best practices

### CODE REVIEW

**Comment by ashwanthkumar** [comment]
👍 This is really awesome!

File: tasks/oracle.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+- name: Automatically select the Oracle License
+  shell: echo debconf shared/accepted-oracle-license-v1-1 select true | sudo debconf-set-selections
+  changed_when: false
+
+- name: Install Oracle Java
+  sudo: yes
+  apt:
+    pkg: "{{item}}"
+    state: latest
```
Comment (arijit-indix) [inline tasks/oracle.yml]: Why this should be `latest` and not `present` ?

File: tasks/openjdk.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+- include: openjdkppa.yml
+
+- name: Install OpenJDK
+  sudo: yes
+  apt:
+    pkg: "{{item}}"
+    state: latest
```
Comment (arijit-indix) [inline tasks/openjdk.yml]: Why this should be `latest` and not `present` ?

File: test_vars/main.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+java_versions:
+  - oracle-jdk7-installer
+  - openjdk-7-jdk
+java_default_version: java-8-oracle
```
Comment (arijit-indix) [inline test_vars/main.yml:4]: Shouldn't these variables go inside defaults/main.yml ? Also, we should pin the exact version.

**Pull request merged**